app/templates/components/project-geometries/draw-lots-to-union.hbs

```diff
@@ -21,11 +21,30 @@
   {{/map.source}}
 {{/if}}
 
+<div class="draw-controls">
+  <button class="polygon {{if (eq drawMode 'draw_polygon') 'active'}}" {{action 'handleDrawButtonClick'}}>{{fa-icon 'draw-polygon' size='2x' fixedWidth=true}}</button>
+  <button class="trash" {{action 'handleTrashButtonClick'}}>{{fa-icon 'trash-alt' prefix='far' size='2x' fixedWidth=true}}</button>
+</div>
```

Comment:
What do you think about moving this to a separate component? We could pass it `map` for the mode state, `draw` for the mode state, and a closure action that is fired when `modechange` occurs.

Reply:
If you think it's necessary. Is it really bloating the `draw-lots-to-union` all that much? What's the advantage of separating it if it won't be used in other places?

Reply:
I don't think _reuse_ is the main benefit of separating out concerns - it's just good practice, easier to reason about, and makes things easier to test out.